README.md

```diff
@@ -75,6 +76,41 @@ methods:
 For definitions for the various fields, consult the corresponding template file for the type of data you are updating.
 
 
+### How to Register Downtime ###
+
+Downtime is a period of time for which one or more services you provide are unavailable.
+You should register downtime if one of these is true:
+
+-  your site is part of the WLCG
+-  your CE is one of the services affected
+
+1. The file that contains downtime information about resources you own is named
```

Comment:
This should start with something like "To register a downtime, identify the file..."

https://opensciencegrid.org/technology/documentation/writing-documentation/#tips-for-writing-procedural-instructions